friture/controlbar.py

```diff
@@ -45,6 +45,18 @@ def __init__(self, parent):
         settings_icon.addPixmap(QtGui.QPixmap(":/images-src/dock-settings.svg"), QtGui.QIcon.Normal, QtGui.QIcon.Off)
         self.settings_button.setIcon(settings_icon)
 
+        self.order_previous = QtWidgets.QToolButton(self)
```

Comment:
Could you please use 'move' everywhere instead of 'order', for consistency with dock.py? I prefer 'move'